inc/commonitilobject.class.php

```diff
@@ -8239,4 +8239,31 @@ public static function getOpenCriteria(): array {
          ]
       ];
    }
+
+   public function displayHiddenItemsIdInput(array $options): void {
+      foreach ($options['items_id'] ?? [] as $itemtype => $items) {
```

Comment:
On a new problem form from a fresh install, I get:
```
[16-Oct-2020 06:15:23 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/webapps/glpi/inc/commonitilobject.class.php on line 8244
[16-Oct-2020 06:15:23 UTC] PHP Stack trace:
[16-Oct-2020 06:15:23 UTC] PHP   1. {main}() /var/www/webapps/glpi/ajax/common.tabs.php:0
[16-Oct-2020 06:15:23 UTC] PHP   2. CommonGLPI::displayStandardTab() /var/www/webapps/glpi/ajax/common.tabs.php:92
[16-Oct-2020 06:15:23 UTC] PHP   3. Problem->showForm() /var/www/webapps/glpi/inc/commonglpi.class.php:627
[16-Oct-2020 06:15:23 UTC] PHP   4. Problem->displayHiddenItemsIdInput() /var/www/webapps/glpi/inc/problem.class.php:1208
```

Reply:
@cedric-anne reported it but I missed it.
Should be good now, fixed in a0fabc3.